src/components/MusicList/index.jsx

```diff
@@ -1,5 +1,5 @@
 const MusicList = () => {
-  return <div></div>;
+  return <p>I'm a musiclist</p>;
```

Comment:
Not a most, but you could just write `Musicalist`. Would make more sense in a real project.

Reply:
Thank that was a reaaaally good point